runtime/doc/eval.txt

```diff
@@ -2477,6 +2478,28 @@ call({func}, {arglist} [, {dict}])			*call()* *E699*
 		{dict} is for functions with the "dict" attribute.  It will be
 		used to set the local variable "self". |Dictionary-function|
 
+capture({command})					*capture()*
+		Capture output of {command}.
+		If {command} is a |String|, it returns {command} output.
+		If {command} is a |List|, it returns concatenated outputs of
+		each item.
+		Examples: >
+			let a = capture('echon "foo"')
+			echo a
+<			foo >
+			let a = capture(['echon "foo"', 'echon "bar"'])
+			echo a
+<			foobar
+		This function is not available in the |sandbox|.
+		Note: The commands are run as if they were prepended with
+		|:silent| modifier.  |:redir| and |capture()| work in parallel:
+		|capture()| does not cancel |:redir|, |:redir| does not cancel
+		|capture()|.  |capture()| inside |capture()| is allowed
+		unlike nested |:redir|s, outer capture will not catch
```

Comment:
It depends on which are considered the independent statements.  Is it `capture() inside capture() is allowed, unlike nested |:redir|s. ...` or `capture() inside capture() is allowed. Unlike nested |:redir|s, ...`.

Also, the phrase `but inner will neither cancel outer` should probably be reworded to something like `but the inner capture will not cancel the outer`.